cmd/sourcepods-dev/dev.go

```diff
@@ -63,6 +63,11 @@ var (
 			Usage: "Storage's root to write to",
 			Value: "./dev/storage-data",
 		},
+		cli.StringFlag{
+			Name:  "ssh-addr",
+			Usage: "The address to run ssh on",
+			Value: ":2222",
```

Comment:
So far I've tried to have the ports > 3000 so something like `30xx` but we can move away from that in this case, maybe. Just thinking out loud.